Framework/Algorithms/test/CalMuonDetectorPhasesTest.h

```diff
@@ -124,6 +124,22 @@ class CalMuonDetectorPhasesTest : public CxxTest::TestSuite {
     runExecutionTest(ws);
   }
 
+  /// Test what happens when you supply frequency in Mrad/s rather than MHz
+  void testFrequencyUnits() {
+    auto ws = createWorkspace(4, 100, "Microseconds");
+    auto calc = AlgorithmManager::Instance().create("CalMuonDetectorPhases");
+    calc->initialize();
+    calc->setChild(true);
+    calc->setProperty("InputWorkspace", ws);
+    calc->setPropertyValue("Frequency", "25"); // Mrad/s, not MHz
+    calc->setPropertyValue("DataFitted", "fit");
+    calc->setPropertyValue("DetectorTable", "tab");
+    calc->setProperty("ForwardSpectra", std::vector<int>{1, 2});
+    calc->setProperty("BackwardSpectra", std::vector<int>{3, 4});
+
+    TS_ASSERT_THROWS(calc->execute(), std::runtime_error);
```

Comment:
Ok, I've removed this test in #16662 because it seems to depend on the machine it runs on - at a later time I can come back and add a more robust test.